dali/rfs/rfs.cpp

```diff
@@ -163,6 +163,9 @@ enum {
     RFCcopysection,
     RFCtreecopy,
     RFCtreecopytmp,
+    RFCsetthrottle,
+    RFCsetthrottle2,
+    RFCsetfileperms,
```

Comment:
added RFCsetthrottle and RFSsetthrottle2 to keep in sync with sockfile.cpp.
Perhaps in the future we should declare these in one place.